client/src/app/pages/review/components/review-form/review-form.tsx

```diff
@@ -107,39 +112,11 @@ export const ReviewForm: React.FC<IReviewFormProps> = ({
     console.log("Invalid form", errors);
   };
 
-  // const onSubmit = (formValues: FormValues) => {
-  //   const payload: Review = {
-  //     ...review,
-  //     proposedAction: formValues.action,
-  //     effortEstimate: formValues.effort,
-  //     businessCriticality: formValues.criticality || 0,
-  //     workPriority: formValues.priority || 0,
-  //     comments: formValues.comments.trim(),
-  //     // application: { ...application, review: undefined },
-  //     application: { id: application.id, name: application.name },
-  //   };
-
-  //   let promise: Promise<Review>;
-  //   if (review) {
-  //     promise = updateReview({
-  //       ...review,
-  //       ...payload,
-  //     });
-  //   } else {
-  //     promise = createReview(payload);
-  //   }
-
-  //   promise
-  //     .then((response) => {
-  //       onSaved(response);
-  //     })
-  //     .catch((error) => {});
-  // };
   const queryClient = useQueryClient();
   const onHandleUpdateReviewSuccess = () => {
     queryClient.invalidateQueries([
       reviewsByItemIdQueryKey,
-      application.review?.id,
+      application?.review?.id,
```

Comment:
Would it make more sense to push the query invalidation to the `updateReviewMutation` itself instead of leaving it here?